src/js/providers/html5.js

```diff
@@ -198,6 +198,9 @@ define([
             }
             _setAudioTracks(_videotag.audioTracks);
             _setTextTracks(_videotag.textTracks);
+            if (_videotag.textTracks.length) {
```

Comment:
Put this in `_setSubtitlesTrack` right before setting `mode = "showing"`, if and only if the textTrack has a crossorigin src attribute (`utils.crossdomain(texttrack.src)`).

Clear the attribute in `_setVideotagSource`. And on stop?